packages/grafana-ui/src/components/PanelChrome/PanelChrome.story.tsx

```diff
@@ -105,4 +109,48 @@ export const Examples = () => {
   );
 };
 
+export const Basic: ComponentStory<typeof PanelChrome> = (args: PanelChromeProps) => {
+  const theme = useTheme2();
+
+  const contentStyle: CSSProperties = {
+    background: theme.colors.background.secondary,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+  };
+
+  return (
+    <PanelChrome {...args}>
+      {(width: number, height: number) => <div style={{ height, width, ...contentStyle }}>Description text</div>}
+    </PanelChrome>
+  );
+};
+
+const Default = <></>;
+const LoadingIcon = <PanelChrome.LoadingIndicator loading onCancel={action('LoadingIndicator: onCancel fired')} />;
+const ErrorIcon = <PanelChrome.ErrorIndicator error="Error text" onClick={action('ErrorIndicator: onClick fired')} />;
```

Comment:
> when there is no title, the leftItems slightly overflow the panel chrome component
I think this is because the `leftItems` are assumed to be an array and we check if `leftItems.length > 0` to calculate the header size, so if we make these two options arrays then that should work!

```suggestion
const LoadingIcon = [
  <PanelChrome.LoadingIndicator key="loading" loading onCancel={action('LoadingIndicator: onCancel fired')} />,
];
const ErrorIcon = [
  <PanelChrome.ErrorIndicator key="error" error="Error text" onClick={action('ErrorIndicator: onClick fired')} />,
];```